lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -521,7 +519,15 @@ def test_user_groups(self):
         mock_user.is_authenticated.return_value = False
         self.assertEqual(views.user_groups(mock_user), [])
 
+    # TODO: TNL-6546: Remove decorator for unified_course_view
+    @override_flag('unified_course_view', active=True)
     def test_get_redirect_url(self):
+        # test the course location
+        self.assertIn(
+            'activate_block_id',
+            get_redirect_url(self.course_key, self.course.location),
```

Comment:
I'm thinking you could also verify that the correct course key is in the URL too. Does that make sense?